util/types/src/extension/serialized_size.rs

```diff
@@ -72,3 +72,17 @@ impl<'r> packed::BlockReader<'r> {
     }
 }
 impl_serialized_size_for_entity!(Block, serialized_size_without_uncle_proposals);
+
+impl packed::UncleBlock {
+    /// Calculates the serialized size of a UncleBlock in Block.
+    pub fn serialized_size_in_block() -> usize {
+        packed::Header::TOTAL_SIZE + 5 * molecule::NUMBER_SIZE
```

Comment:
`UncleBlock` has one `proposals` field, did you miss the calculation here? And how the hardcoded `5` number comes from?

Reply:
We don't calculate uncle's proposals in block size. see `BlockView::serialized_size_without_uncle_proposals` and test `block_size_should_not_include_uncles_proposals`.

    /// The block has 1 more uncle:
    /// - the block will has 1 more offset (+NUM_SIZE) in UncleBlockVec
    /// - UncleBlockVec has 1 more UncleBlock.
    ///      UncleBlock comes with 1 `total` field, and 2 field offsets, (+NUM_SIZE * 3)
    ///      UncleBlock contains Header (+208) and empty proposals (only one total_size, + NUM_SIZE because it is a fixVec)
    /// The total is +NUM_SIZE*5 + Header.size() = 228

comment updated.